source/replication.rst

```diff
@@ -0,0 +1,141 @@
+.. _operator-replication:
+
+Set up Percona XtraDB Cluster cross-site replication
+====================================================
+
+The cross-site replication involves configuring one Percona XtraDB Cluster as *Source*, and another Percona XtraDB Cluster as *Replica* to allow an asynchronous replication between them:
+
+ .. image:: ./assets/images/pxc-replication.svg
+   :align: center
+
+The Operator automates configuration of *Source* and *Replica* Percona XtraDB Clusters, but the feature itself is not bound to Kubernetes. Either *Source* or *Replica* can run outside of Kubernetes and be out of the Operators’ control. 
```

Comment:
```suggestion
The Operator automates configuration of *Source* and *Replica* Percona XtraDB Clusters, but the feature itself is not bound to Kubernetes. Either *Source* or *Replica* can run outside of Kubernetes, be regular MySQL and be out of the Operators’ control. 
```

Reply:
I believe it is important to mention that this feature can work for regular MySQL as well, not only PXC.